src/prop13/graphlibrary/Edge.java

```diff
@@ -5,13 +5,33 @@
  */
 public abstract class Edge
 {
-    protected Node n1;
-    protected Node n2;
-    protected int weight;
+    private Node n1;
+    private Node n2;
+    private int weight;
 
-    public Edge()
-    {
-    }
+	public Node getN1() {
```

Comment:
Yea, I get it. It's still only beginning. But I don't know what other teams need. We can implement undirected in higher layer (relatin, member, congress) But OK I will delete it.